static/src/javascripts/projects/common/modules/analytics/beacon.js

```diff
@@ -15,6 +17,16 @@ define([
 
             return img;
         },
+        postJson: function (path, jsonString) {
+            return ajax({
+                url: config.page.beaconUrl + path,
+                type: 'json',
+                method: 'post',
+                contentType: 'application/json',
+                data: jsonString,
+                crossOrigin: true
+            });
+        },
```

Comment:
Would be nice to use the [Beacon API](https://developer.mozilla.org/en-US/docs/Web/API/navigator.sendBeacon) for this if available, I.e something like:
```js 
if('sendBeacon' in navigator) { }
```